index.bs

```diff
@@ -8230,8 +8230,8 @@ IDL [=promise type=] values are represented by ECMAScript [=PromiseCapability=]
 
 <div algorithm>
 
-    To <dfn id="dfn-perform-steps-once-promise-is-settled" export lt="upon settling">perform some steps once a promise is settled</dfn>,
-    given a <code><a interface>Promise</a>&lt;|T|&gt;</code> |promise| and one or two sets of steps
+    To <dfn id="dfn-perform-steps-once-promise-is-settled" export lt="react|reacting" for="promise">react</dfn>
```

Comment:
"promise" or "Promise"?

Reply:
"promise" is the concept, `Promise<T>` is the type, so I think lowercase is good here.